addon-test-support/@ember/test-helpers/dom/trigger-key-event.js

```diff
@@ -0,0 +1,54 @@
+import { merge } from '@ember/polyfills';
+import getElement from './-get-element';
+import triggerEvent from './trigger-event';
+import { KEYBOARD_EVENT_TYPES } from './fire-event';
+
+const DEFAULT_MODIFIERS = Object.freeze({
+  ctrlKey: false,
+  altKey: false,
+  shiftKey: false,
+  metaKey: false,
+});
+
+/**
+  @public
+  @param {String|HTMLElement} target
+  @param {'keydown' | 'keyup' | 'keypress'} eventType
+  @param {String} keyCode
+  @param {Object} modifiers
+  @param {Boolean} modifiers.ctrlKey
+  @param {Boolean} modifiers.altKey
+  @param {Boolean} modifiers.shiftKey
+  @param {Boolean} modifiers.metaKey
+  @return {Promise<void>}
+*/
+export default function triggerKeyEvent(target, eventType, keyCode, modifiers = DEFAULT_MODIFIERS) {
+  if (!target) {
+    throw new Error('Must pass an element or selector to `triggerKeyEvent`.');
+  }
```

Comment:
we don't seem to have this check in any of the other helpers above 🤔

Reply:
As far as I can tell all of the helpers throw an error if the `target` / `selectorOrElement` is not provided.  Which once did I miss?

Reply:
e.g. `fillIn` and `tap` do not have this check and delegate to `getElement` directly